src/component/registry/bpmn-model-filters.ts

```diff
@@ -45,9 +45,9 @@ export class ModelFiltering {
     // If there is no shape associated to a Pool, no flow nodes, no lanes and no edges, there is no ShapeBPMNElement associated to the pool id to filter.
     // So we need to throw an error.
     if (filteredPools.length == 0 && filteredLanes.length == 0 && filteredFlowNodes.length == 0 && filteredEdges.length == 0) {
-      let errorMsgSuffix = poolIdsFilter.length > 0 ? ` for ids [${poolIdsFilter}]` : '';
+      let errorMsgSuffix = poolIdsFilter.length > 0 ? ` for ids [${poolIdsFilter.join(',')}]` : '';
```

Comment:
**question:** I guess this is not not depend on the `toString` implementation of an array and make this explicit?